values.yaml

```diff
@@ -103,11 +103,19 @@ global:
         _default: hostpath
         local: hostpath
         aws: efs-sc
+      auto:
+        _default: false
+        local: false
+        aws: false
+      mode:
+        _default: ReadWriteOnce
+        local: ReadWriteMany
+        aws: ReadWriteMany
```

Comment:
These only need to be ReadOnlyMany, unless that still isn't properly supported by the aws drivers?
I'd expect _default and local to be ReadOnlyMany if possible and aws ReadWriteMany if need be

Reply:
In aws it's not supported under EFS. I've made it ReadWriteMany for the local profile because it's become clear that deploying the application on the infrastructure is going to be inflexible in cases where you have heterogeneous storage typologies and may require dynamic provisioning. Going to raise a ticket to provide a rest endpoint for deploying jars onto the storage.